src/ripple/overlay/impl/ProtocolMessage.h

```diff
@@ -83,28 +84,51 @@ struct MessageHeader
 
     /** The type of the message. */
     std::uint16_t message_type = 0;
+
+    /** Indicates whether the message is compressed. */
+    bool compressed = false;
```

Comment:
Now that `Algorithm::None` is added, do we still need this `compressed` member?

Reply:
Removed `compressed`.